src/transformers/models/zoedepth/image_processing_zoedepth.py

```diff
@@ -442,3 +449,129 @@ def preprocess(
+            remove_padding (`bool`, *optional*):
+                By default ZoeDepth addes padding to fix the boundary artifacts in the output depth map, so we need
+                remove this padding during post_processing. The parameter exists here in case the user changed the image
+                preprocessing to not include padding.
+
+            vmin_perc (`float`, *optional*, defaults to `1.0`):
+                use the `vmin_perc`-th percentile as minimum value during normalization (outlier rejection).
+            vmax_perc (`float`, *optional*, defaults to `99.0`):
+                use the `vmax_perc`-th percentile as maximum value during normalization (outlier rejection).
+            normalize (`bool`, *optional*, defaults to `False`):
+                Apply normalization between [0,1] for the colored image values.
+            cmap (`str`, *optional*, defaults to `gray_r`):
+                matplotlib colormap to use (requires matplotlib).
+            gamma_corrected (`bool`, *optional*, defaults to `False`):
+                Apply gamma correction to colored image.
+
+        Returns:
+            `List[Dict]`: A list of dictionaries, each dictionary containing the depth predictions and a depth PIL image
+            as predicted by the model.
+        """
+        requires_backends(self, "torch")
+
+        predicted_depth = outputs.predicted_depth
+
+        if (outputs_flip is not None) and (predicted_depth.shape != outputs_flip.predicted_depth.shape):
+            raise ValueError("Make sure that `outputs` and `outputs_flip` have the same shape")
+
+        if (target_sizes is not None) and (len(predicted_depth) != len(target_sizes)):
+            raise ValueError(
+                "Make sure that you pass in as many target sizes as the batch dimension of the predicted depth"
+            )
+
+        if remove_padding is None:
+            remove_padding = self.do_pad
+
+        if (source_sizes is None and remove_padding) or (len(predicted_depth) != len(source_sizes)):
+            raise ValueError(
+                "Make sure that you pass in as many source image sizes as the batch dimension of the logits"
+            )
+
+        if outputs_flip is not None:
+            predicted_depth = torch.stack([predicted_depth, outputs_flip.predicted_depth], dim=1)
+        else:
+            predicted_depth = predicted_depth.unsqueeze(1)
+
+        # Zoe Depth model adds padding around the images to fix the boundary artifacts in the output depth map
+        # The padding length is `int(np.sqrt(img_h/2) * fh)` for the height and similar for the width
+        # fh (and fw respectively) are equal to '3' by default
+        # Check [here](https://github.com/isl-org/ZoeDepth/blob/edb6daf45458569e24f50250ef1ed08c015f17a7/zoedepth/models/depth_model.py#L57)
+        # for the original implementation.
+        # In this section, we remove this padding to get the final depth image and depth prediction
+        fh = fw = 3
```

Comment:
What do 'fh' and 'fw' stand for?

Reply:
These are parameters from the original implementation and are used in the same file to calculate `pad_h` and `pad_w`. For the implementation in transformers, those parameters have been already set to `3` by default in the pre-processor

Reply:
I meant more what is the "f" in "fh" or the "f" in "fw" -- all the variables should have clear, explicit names e.g. `batch_size` instead of `b`

Reply:
According to the official repository, those are
```Python
fh (float, optional): height padding factor. The padding is calculated as sqrt(h/2) * fh. Defaults to 3.
fw (float, optional): width padding factor. The padding is calculated as sqrt(w/2) * fw. Defaults to 3.
```
I'll rename the variables to padding_factor_h and padding_factor_w to be more descriptive.

Reply:
Thanks!